src/DataTable/column.ts

```diff
@@ -21,6 +24,18 @@ export interface Column<Data extends UniqueRow> {
    */
   field: ObjectPaths<Data>
 
+  /**
+   * The minimum width the column can shrink to
+   */
+  // TODO: uncomment ResponsiveValue<T> when I'm ready to implement the responsive part
```

Comment:
I'm not even sure we need to make these props responsive. Reviewers: how would you feel about leaving out the responsive value feature until we find we actually need it?

Reply:
> Reviewers: how would you feel about leaving out the responsive value feature until we find we actually need it?

Definitely down for this 👍